src/pages/login-page/LoginPage.tsx

```diff
@@ -86,7 +86,7 @@ const LoginPage = () => {
       <Box height="100vh" className={classes.wrapper}>
         <Container maxWidth="xl" component="main" className={classes.container}>
           <Box component="form" noValidate maxWidth="md">
-            <Box component="img" src={SquareGroup} alt="login" className={classes.image} />
+            <Box component="img" src={MetisLogo} alt="login" className={classes.image} />
```

Comment:
I'm watching the result and think so we will change to svg format or put a image with major quality. The image looks up pixeled